src/lib.rs

```diff
@@ -103,7 +104,7 @@ impl Dex {
 }
 
 impl From<DexReader> for Dex {
-    fn from(reader: DexReader) -> Dex {
+    fn from(_: DexReader) -> Dex {
```

Comment:
Actually, I like this warning, since it remembers what it's still not implemented.